packs/spell-effects/aura-silence-heightened-4th.json

```diff
@@ -0,0 +1,49 @@
+{
+    "_id": "diKAX7qm2M07jnpe",
+    "img": "systems/pf2e/icons/spells/silence.webp",
+    "name": "Aura: Silence (Heightened 4th)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Silence]</p>\n<p>The spell creates an aura in a 10-foot emanation around the touched creature, silencing all sound in or passing through it. While within the aura, creatures are subject to the same effects as the target.</p>"
```

Comment:
Maybe we want to do this all as one effect? Have the level 4 effect just grant level 1 copies of itself?